client/my-sites/earn/home.tsx

```diff
@@ -154,18 +156,37 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 					},
 			  }
 			: {
-					text: translate( 'Collect recurring payments' ),
+					text: isPayments
+						? translate( 'Collect payments' )
+						: translate( 'Collect recurring payments' ),
 					action: () => {
 						trackCtaButton( 'recurring-payments' );
 						page( `/earn/payments/${ selectedSiteSlug }` );
 					},
 			  };
-		const title = hasConnectedAccount
-			? translate( 'Manage recurring payments' )
+		const hasConnectionTitle = isPayments
+			? translate( 'Manage payments' )
+			: translate( 'Manage recurring payments' );
+		const noConnectionTitle = isPayments
+			? translate( 'Collect payments' )
 			: translate( 'Collect recurring payments' );
-		const body = hasConnectedAccount
+		const title = hasConnectedAccount ? hasConnectionTitle : noConnectionTitle;
+
+		const hasConnectionBody = isPayments
 			? translate(
+					"Manage your customers and subscribers, or your current subscription options and review the total revenue that you've made from payments."
+			  )
+			: translate(
 					"Manage your subscribers, or your current subscription options and review the total revenue that you've made from recurring payments."
+			  );
+		const noConnectionBody = isPayments
+			? translate(
+					'Collect payments / Accept one-time and recurring credit card payments for digital goods, physical products, services, memberships, subscriptions, and donations. {{em}}Available with any paid plan{{/em}}.',
```

Comment:
I think this reads a bit awkward. Since "Collect payments" is already part of the title, can we maybe just drop it?

```suggestion
					'Accept one-time and recurring credit card payments for digital goods, physical products, services, memberships, subscriptions, and donations. {{em}}Available with any paid plan{{/em}}.',
```

Reply:
I think tightening copy here is a good call!